src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs

```diff
@@ -9,7 +9,10 @@ public static class KaikoClientRegistration
     public static IServiceCollection AddKaikoClient(this IServiceCollection services, IConfiguration configuration)
     {
         var config = configuration.GetSection(nameof(KaikoApiConfiguration)).Get<KaikoApiConfiguration>();
-        services.AddKaikoClient(config);
+        if (config != null)
```

Comment:
Shouldn't this throw an exception if the config doesn't exist?

Reply:
We don't have that behaviour in other api clients. In other clients, we just assume the config is there. Here I merely tried to silence a warning. I can revert if it's easier to pass the PR 😅

Reply:
<https://github.com/trakx/coingecko-api-client/blob/dev/src/Trakx.CoinGecko.ApiClient/AddCoinGeckoClientExtension.cs#L20>

Reply:
<https://github.com/trakx/cryptocompare-api-client/blob/dev/src/Trakx.CryptoCompare.ApiClient/ServiceConfiguration.cs>

(examples of what I mean)

Reply:
It's fixed, `!` and done :)

Reply:
Nice :)